apps/namadillo/src/App/Governance/ProposalLabels.tsx

```diff
@@ -24,22 +24,35 @@ export const StatusLabel: React.FC<
   );
 };
 
-export const VotedLabel: React.FC<React.ComponentProps<"div">> = ({
+type VotedLabelProps = {
+  vote: VoteType;
+} & React.ComponentProps<"div">;
+
+export const VotedLabel: React.FC<VotedLabelProps> = ({
+  vote,
   className,
   ...props
 }) => {
+  const { icon, label, color } =
+    vote === "yay" ?
+      { icon: <GoCheckCircleFill />, label: "yes", color: "text-yay" }
+    : vote === "nay" ?
+      { icon: <GoXCircleFill />, label: "no", color: "text-nay" }
+    : vote === "abstain" ?
+      {
+        icon: <GoSkipFill className="rotate-45" />,
```

Comment:
I couldn't see a suitable icon for this so I used rotate, but if there is a better way that would be good to know.